tests/rptest/archival/s3_client.py

```diff
@@ -61,9 +61,12 @@ def __init__(self,
     def create_bucket(self, name):
         """Create bucket in S3"""
         try:
-            self._cli.create_bucket(
+            res = self._cli.create_bucket(
```

Comment:
If this wasn't checking success before, is it possible that this was the cause of the mysterious NoSuchBucket errors?

Reply:
Unfortunately no